environments/icds-cas/postgresql.yml

```diff
@@ -5,7 +5,6 @@ REPORTING_DATABASES:
   icds-ucr:
     WRITE: icds-ucr
     READ:
-      - [icds-ucr, 1]
```

Comment:
if all of these standbys are behind, would the dashboard and UCRs not load since there wouldn't be any relevant READ databases? Looks like the "acceptable delay" is 30 min at the moment. Not sure how close we get to that